src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -355,6 +351,23 @@ public AgentSettings LoadSettings()
             var serviceControlManager = HostContext.GetService<ILinuxServiceControlManager>();
```

Comment:
@kmkumaran these two lines are in the #elif block. So, they won't execute now too.